transport_nantes/mobilito/views.py

```diff
@@ -166,13 +174,21 @@ class AddressFormView(LoginRequiredMixin, FormView):
     success_url = reverse_lazy('mobilito:recording')
 
     def get_context_data(self, **kwargs):
-        context = super().get_context_data(**kwargs)
+        context = super(FormView, self).get_context_data(**kwargs)
         # Clear session data if user had filled the form before.
         self.request.session['address'] = None
         self.request.session['city'] = None
         self.request.session['postcode'] = None
         return context
 
+    def get(self, request, *args, **kwargs):
+        context = self.get_context_data(**kwargs)
+        if self.get_pages_to_visit():
+            return HttpResponseRedirect(
```

Comment:
Unless you mean that the client's browser may, the next time, assume the same redirect, you should make sure you 302.

Indeed, specifying what you want instead of relying on the default value makes your code more robust to future changes in underlying libraries (including accidental ones).

Reply:
HttpResponseRedirect is a 302.